addons/mail/static/src/core/common/out_of_focus_service.js

```diff
@@ -53,10 +53,10 @@ export class OutOfFocusService {
                 notificationTitle = author.name;
             }
         }
-        const notificationContent = htmlToTextContentInline(message.body).substring(
-            0,
-            PREVIEW_MSG_MAX_SIZE
-        );
+        const messageBodyText = message.hasOnlyAttachments
+            ? message.messagePreviewText
+            : htmlToTextContentInline(message.body);
+        const notificationContent = messageBodyText.substring(0, PREVIEW_MSG_MAX_SIZE);
```

Comment:
Can be simplified to:
```js
const notificationContent = message.previewText.substring()
```
As `message.previewText` covers both message with and without text.